src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexFacetedReadOperation.cs

```diff
@@ -123,6 +125,40 @@ void UpdateRangeResults()
                 }
             }
         }
+        
+        void CreateMappingForRanges()
+        {
+            foreach (var result in results)
+            {
+                if (result.Value.Ranges == null || result.Value.Ranges.Count == 0)
+                    continue;
+
+                // Cache facetByRange because we will fulfill data in batches instead of whole collection
+                if (facetsByRange.TryGetValue(result.Key, out var facetValues) == false)
+                {
+                    facetValues = new();
+                    facetsByRange.Add(result.Key, facetValues);
+                }
+
+                foreach (var range in result.Value.Ranges)
+                {
+                    var key = range.RangeText;
+                    if (facetValues.TryGetValue(key, out var collectionOfFacetValues))
+                        continue;
+
+                    collectionOfFacetValues = new FacetValues(facetQuery.Legacy);
+                    if ((result.Value.Aggregations.Count > 0) == false)
+                        collectionOfFacetValues.AddDefault(key);
```

Comment:
Just stylistic, if you have braces on the else... would make sense to add braces on the if too.